src/OceanSeaIceModels/InterfaceComputations/assemble_net_fluxes.jl

```diff
@@ -261,8 +261,8 @@ end
     @inbounds begin
         Ts = surface_temperature[i, j, kᴺ]
         Ts = convert_to_kelvin(sea_ice_properties.temperature_units, Ts)
-        ℵi = ice_concentration[i, j, 1]
-        
+        ℵi = ice_concentration[i, j, kᴺ]
```

Comment:
is this a bug that exists on main? let's fix it in its own PR because it may be a while until this PR is merged?

Reply:
see  #561